# Note that this must be loaded in the namespace of the node's name
gas_diffusivity: 10.
gas_diff_rate: 1.
gas_ttl: 1.
wind_angle: -0.8726
wind_velocity: 1.
sensor_radius: 1
delta_t: 1
resolution: 5.
fast_infotaxis: false

max_stepdist: 10
med_stepdist: 1

min_stepnumber: 24
stepnumber: 4
max_stepnumber: 200

publish_backtrace: true
print_images: true
detect_while_moving: true
detect_while_moving: 4
run_rate: 1
#simulate_source: false
simulate_source: true
source_x: -7.83
source_y: 14.21
goal_tolerance: .2
